ui/board.controller.js

```diff
@@ -11,14 +11,21 @@
     vm.minDataPoints = 10;
     vm.maxDataPoints = 1000;
 
-    vm.windowOpts = Util.toDurations(['1.4 years', '12 months', '8 months', '6 months', '4 months', '2 months', '1 month', '2 weeks', '10 days', '1 week', '6 days', '5 days', '4 days', '3 days',
+    vm.windowOpts = Util.toDurations(['15 months', '12 months', '8 months', '6 months', '4 months', '2 months', '1 month', '2 weeks', '10 days', '1 week', '6 days', '5 days', '4 days', '3 days',
       '2 days', '1 day', '18 hours', '12 hours', '6 hours', '3 hours', '2 hours', '1 hour']);
-    vm.periodOpts = Util.toDurations(['1 month', '5 days', '10 days', '5 days', '2 days', '1 day', '6 hours', '1 hour', '15 minutes', '5 minutes', '1 minute']);
+    vm.periodOpts = Util.toDurations(['1 month', '15 days', '10 days', '5 days', '2 days', '1 day', '12 hours', '6 hours', '1 hour', '15 minutes', '5 minutes', '1 minute']);
```

Comment:
Concrete suggestion:

Replace:
- '1 month' with '4 weeks'
- '15 days' with '2 weeks'

Abstract rationale:

I suggest the `periodOpts` might be a little better if they topped-out at multiples of weeks.  This synchronizes with our crawling schedule better and other metrics (like the R28 that the business side uses).  Having slightly out-of-phase periods will eventually result in discrepancies when trying to correlate data from different places.  Also, '1 month' is not a consistent measurement unit for recurrences ... it means different things at different times of the year.